app/eligibility/update-waiting.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../data')

```

Comment:
could we call this `process-eligible`? I think alongside the database activity we will be sending emails etc etc from the same function later on